LondonBritishLibrary/orient/BLorient803.xml

```diff
@@ -130,13 +115,103 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                             <citedRange unit="page">280a-281b</citedRange>
                                         </bibl>
                                     </listBibl>
-                                    
                                 </source>
                             </recordHist>
                         </adminInfo>
                     </additional>
+
+<msPart xml:id="p1">
+<msIdentifier></msIdentifier>
+<msContents>
+<msItem xml:id="p1_i1">
+<locus from="2r"/>
+<title xml:lang="gez" type="complete" ref="LIT1395Fethan">Fǝtḥa nagaśt</title>
+<textLang mainLang="gez"/>
+
+<msItem xml:id="p1_i1.1">
+<locus from="2r"/>
+<title xml:lang="en" type="complete" ref="LIT1395Fethan#EthiopicIntroduction">Introduction</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.2">
+<locus from="6r"/>
+<title xml:lang="gez" type="complete" ref="LIT1395Fethan#IntroductionCanons">Table of contents</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.3">
+<locus from="6r"/>
+<title xml:lang="en" type="complete" ref="LIT1395Fethan#Part1">The first part <foreign xml:lang="gez">
+ክፍል፡ ቀዳማዊ፡</foreign>Ecclesiastical law</title>
+<incipit xml:lang="gez">አንቀጽ፡ ቀዳማዊ፡ በእንተ፡ ቤተ፡ ክርስቲያን፡ ወዘይደሉ፡ ላቲ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i1.4">
+<locus from="66r"/>
+<title xml:lang="en" type="complete" ref="LIT1395Fethan#Part2">The second part <foreign xml:lang="gez">
+ክፍል፡ ዳግማዊ፡</foreign>Civil law</title>
+<incipit xml:lang="gez">አንቀጽ፡ ፳ወ፫፡ በእንተ፡ መብልዕ፡ ወአልባስ፡ ወማኅደር፡ ወተግባረ፡ እድ፡ ዘይደሉ፡ ለመሲሐውያን።</incipit>
```

Comment:
please, add a white space befoer Civil here as well. Basically this information wight also be added to the work record and then  skipped here